blocks/init/src/Blocks/custom/tab/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+	"blockName": "tab",
+	"title": "Tab",
+	"description" : "Tab block with custom settings.",
+	"category": "eightshift",
+	"icon": {
+		"src": "es-column"
```

Comment:
I think @goranalkovic-infinum should create a cool icon for this 😄 

#delegation